docs/docs/reference-react-component.md

```diff
@@ -186,24 +240,36 @@ If `props.color` is set to null, it will remain null:
   }
 ```
 
+* * *
+
 ### `displayName`
 
 The `displayName` string is used in debugging messages. JSX sets this value automatically; see [JSX in Depth](/react/docs/jsx-in-depth.html).
 
-### `forceUpdate(callback)`
+* * *
+
+### `forceUpdate()`
+
+```javascript
+component.forceUpdate(callback)
+```
 
 By default, when your component's state or props change, your component will re-render. If your `render()` method depends on some other data, you can tell React that the component needs re-rendering by calling `forceUpdate()`.
 
 Calling `forceUpdate()` will cause `render()` to be called on the component, skipping `shouldComponentUpdate()`. This will trigger the normal lifecycle methods for child components, including the `shouldComponentUpdate()` method of each child. React will still only update the DOM if the markup changes.
 
 Normally you should try to avoid all uses of `forceUpdate()` and only read from `this.props` and `this.state` in `render()`.
 
+* * *
+
```

Comment:
There is a confusion here between properties available on an instance (props and state) and properties you need to set on the class (defaultProps, displayName). Can you please separate props and state together rather than have them mixed with class properties? Maybe group them as "Instance Fields" and group the rest as "Class Fields"?